internal/pkg/mapper/orchestrator/remote_load.go

```diff
@@ -219,6 +211,14 @@ func (l *remoteLoader) parseTask(taskRaw interface{}) error {
 		}
 	}
 
+	// Get target config
+	targetConfig, err := l.getTargetConfig(task.ComponentId, task.ConfigId)
+	if err != nil {
+		errors.Append(err)
+	} else if targetConfig != nil {
+		markConfigUsedInOrchestrator(targetConfig, l.config)
```

Comment:
To iste aj v `remote load` + tu sa vobec nekontrolovalo, ze config pouzity v orchestracii existuje -> fixnute.